netbox_attachments/tests/fixtures/swap_netbox_config.sh

```
@@ -0,0 +1,171 @@
+    echo "Usage: $0 <config_number> [--no-backup]"
+    echo ""
+    echo "Available configurations:"
+    ls -1 "$CONFIG_DIR"/config_*.py | sed 's/.*config_/  Config /' | sed 's/_.*//'
+    exit 1
+fi
+
+CONFIG_NUM=$1
+NO_BACKUP=$2
+
+# Find config file with glob expansion
+CONFIG_FILE=$(ls "$CONFIG_DIR"/config_$(printf "%02d" "$CONFIG_NUM")_*.py 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)